configure

```diff
@@ -26,10 +26,16 @@
 # make sure that is called using bash.
 
 # Get an absolute path to this script, since that determines the top-level directory.
-this_script_dir=`dirname $0`
-this_script_dir=`cd $this_script_dir > /dev/null && pwd`
+source_path="$(dirname ${0})"
+
+this_script_dir="$(cd -- "${source_path}" > /dev/null && pwd)"
+
+if [ -z "${this_script_dir}" ]; then
+	echo "# Could not determine location of configure script"
```

Comment:
Why the initial `#`?

Reply:
Just habit. Some style guides for shell commands suggest putting a "#" before error output.